metabase/metabase
# Fix can't select dashboard tabs in interactive embedding with parameter `header=false`

> [!note]
> This PR supersedes #39005

Closes #38429, #39002

### Description

The [`header` documentation](https://www.metabase.com/docs/latest/embedding/interactive-embedding#header) indicates
> To hide a question or dashboard’s title, additional info, and action buttons

Dashboard tabs is a very crucial part of the dashboard and should, in my opinion, never been hidden.

This PR fixes the problem by only hiding the other parts in the dashboard header except the tabs

**Note** Please also see the #39005 explaining the fix for the dashboard card not loading which is also fixed in this PR with a different approach.

### How to verify
0. Set up interactive embedding
1. Create a dashboard with multiple tabs
2. Visit the interactive embedding page where the iframe URL has this parameter `?header=false`
3. Visit the dashboard we created in 1)
4. See that the tabs are visible and you can select each tab.

### Demo

#### Before
![image](https://github.com/metabase/metabase/assets/1937582/ad5e9952-0f64-44fe-a9e1-dbb0538962d9)


#### After
![image](https://github.com/metabase/metabase/assets/1937582/89c0d127-b8e0-4f76-967f-80f92148beb4)


### Checklist
